### PR TITLE
feat: added log event emitting

### PR DESCRIPTION
**Pull request by chexware**

### CODE REVIEW

**Review by librelois — APPROVED** [review]

File: src/tracing.rs [hunk 1]
```diff
@@ -67,6 +68,11 @@ pub enum Event<'a> {
 		is_static: bool,
 		context: &'a Context,
 	},
+	Log {
+		address: H160,
+		topics: Vec<H256>,
+		data: Vec<u8>,
```
Comment (RomarQ) [inline src/tracing.rs]: To be consistent with other events
```suggestion
		data: &'a [u8],
```

**Review by RomarQ — APPROVED** [review]

File: src/tracing.rs (hunk 1 above)
Comment (chexware) [reply src/tracing.rs]: Updated in https://github.com/moonbeam-foundation/evm/pull/4/commits/ba040f5a16a6e51f06610bc0eaf56bffde2c8a89

**Comment by dnjscksdn98** [comment]
It seems that this PR is not applied to upstream. Currently, frontier uses branch "v0.x" for evm. Is there any purpose that it's only merged to moonbeam?

**Comment by RomarQ** [comment]
> It seems that this PR is not applied to upstream. Currently, frontier uses branch "v0.x" for evm. Is there any purpose that it's only merged to moonbeam?

Opened a pull request upstream: https://github.com/rust-ethereum/evm/pull/330

**Pull request merged**